pyvista/core/filters/data_set.py

```diff
@@ -2068,12 +2070,16 @@ def compute_cell_sizes(self, length=True, area=True, volume=True, progress_bar=F
         progress_bar : bool, default: False
             Display a progress bar to indicate progress.
 
+        vertex_count : bool, default: False
+            Specify whether or not to compute the number of vertices
+            of 0D cells.
```

Comment:
I rewrite the docstring based on the [SetComputeVertexCount](https://vtk.org/doc/nightly/html/classvtkCellSizeFilter.html#af459249cf230a59304e86fbaf9503099) manual. If my understanding is correct, please approve.
```suggestion
            Specify whether or not to compute sizes for vertices and 0D cells.
            The computed value is the number of points in the cell.
```